flutter_local_notifications/example/lib/main.dart

```diff
@@ -982,6 +989,8 @@ class _HomePageState extends State<HomePage> {
       importance: Importance.max,
       priority: Priority.high,
       ticker: 'ticker',
+      color: Colors.blue,
```

Comment:
was there a reason for making this change? It looks like this change has had no effect on the result so i'm guessing it was from your experiments and should be removed